src/Context/Awareness/CommentAwareContextTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace PaulGibbs\WordpressBehatExtension\Context\Awareness;
+
+trait CommentAwareContextTrait
+{
+
+    /**
+     * Create a comment.
+     *
+     * @param array $args
+     *            Set the values of the new comment.
+     *
+     * @return array {
+     *         @type int $id Content ID.
```

Comment:
Indentation for this sort of thing looks a bit off. Not sure if this line needs indenting one more, or if the closing array on the subsequent line that needs to go back one. ?